www/src/components/navigation.js

```diff
@@ -162,6 +162,7 @@ const Navigation = ({ pathname }) => {
           <SocialNavItem
             href="https://www.gatsbyjs.com"
             title="gatsbyjs.com"
+            aria-label="gatsbyjs.com"
             overrideCSS={{ paddingRight: 0, fontSize: presets.scale[1] }}
           >
             .com
```

Comment:
@dengeist - that was you suggestion

```suggestion
             Gatsbyjs.com
```

Reply:
Afaik the `title` is already used then. It is either label or title used.